pkg/serviceaccount/util.go

```diff
@@ -33,25 +33,26 @@ const (
 )
```

Comment:
util_test.go is updated with multi-tenancy change.

Reply:
what do you mean here?

Reply:
Sorry, I mean, util_test.go is NOT updated.

Reply:
updated tests